fmrest/server.py

```diff
@@ -371,7 +371,8 @@ def get_record(self, record_id: int, portals: Optional[List[Dict]] = None,
                    scripts: Optional[Dict[str, List]] = None,
                    layout: Optional[str] = None,
                    request_layout: Optional[str] = None,
-                   response_layout: Optional[str] = None) -> Record:
+                   response_layout: Optional[str] = None,
+                    dateformats: Optional[str] = None) -> Record:
```

Comment:
Ok to use `date_format` parameter if you prefer. I just use same name to avoid typing error ;-)

If we use keywords, we also need to had logic to convert them.
Is this code a way to to that ?

```
API_DATE_FORMATS = [('us', '0'), ('file', '1'), ('iso-8601', '2')]

date_format = 'file'

matching_format = next((format[1] for format in API_DATE_FORMATS if format[0] == date_format), None)
if matching_format:
    params['dateformats'] = matching_format
```